util/req/reqSchema.go

```diff
@@ -0,0 +1,82 @@
+package req
+
+import (
+	"fmt"
+	"io"
+	"reflect"
+	"strings"
+)
+
+const (
+	schemaIndent = "\t"
+)
+
+func GetSchema(dst io.Writer, typ reflect.Type) error {
```

Comment:
The variable name `dst` does not make sense just like this. You need to add a comment to make it more explicit

Reply:
`dst` is quite standard an abbreviation, and with an `io.Writer` seems fairly obvious, but I did forget to document the function.